data/mods/BombasticPerks/perks.json

```diff
@@ -0,0 +1,173 @@
+    "points": 0,
+    "description": "You're like a walking tank!  Take 10% less damage from all sources.",
+    "category": [ "perk" ],
+    "enchantments": [
+      {
+        "condition": "ALWAYS",
+        "values": [
+          { "value": "ARMOR_ACID", "multiply": -0.1 },
+          { "value": "ARMOR_BASH", "multiply": -0.1 },
+          { "value": "ARMOR_BIO", "multiply": -0.1 },
+          { "value": "ARMOR_BULLET", "multiply": -0.1 },
+          { "value": "ARMOR_COLD", "multiply": -0.1 },
+          { "value": "ARMOR_CUT", "multiply": -0.1 },
+          { "value": "ARMOR_ELEC", "multiply": -0.1 },
+          { "value": "ARMOR_HEAT", "multiply": -0.1 },
+          { "value": "ARMOR_STAB", "multiply": -0.1 }
+        ]
+      }
+    ]
+  },
+  {
+    "type": "mutation",
+    "id": "perk_hauler",
+    "name": { "str": "Hauler" },
+    "points": 0,
+    "description": "All that hauling has paid off.  You can carry 10kg of additional items.",
+    "category": [ "perk" ],
+    "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "CARRY_WEIGHT", "add": 10000 } ] } ]
+  },
+  {
+    "type": "mutation",
+    "id": "perk_vacationer",
+    "name": { "str": "Vacationer" },
+    "points": 0,
+    "description": "Every day in the apocalypse is just like the beach.  You are more comfortable in cold and warm weather.",
+    "category": [ "perk" ],
+    "enchantments": [
+      {
+        "condition": "ALWAYS",
+        "values": [ { "value": "CLIMATE_CONTROL_HEAT", "add": 10 }, { "value": "CLIMATE_CONTROL_CHILL", "add": 10 } ]
+      }
+    ]
+  },
+  {
+    "type": "mutation",
+    "id": "perk_thick_skull",
+    "name": { "str": "Thick Skull" },
+    "points": 0,
+    "description": "You've taken so many blows to the head you've lost count.  You've got additional skull armor.",
+    "category": [ "perk" ],
+    "integrated_armor": [ "integrated_thick_skull" ]
+  },
+  {
+    "type": "mutation",
+    "id": "perk_tuck_and_roll",
+    "name": { "str": "Tuck and Roll" },
+    "points": 0,
+    "description": "At some point you got really good at falling off rooftops.  Take 40% less fall damage.",
+    "category": [ "perk" ],
+    "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "FALL_DAMAGE", "multiply": -0.4 } ] } ]
+  },
+  {
+    "type": "mutation",
+    "id": "perk_bloody_mess",
+    "name": { "str": "Bloody Mess" },
+    "points": 0,
+    "description": "For whatever reason you seem to always make a real mess of things.  Enemies you kill tend to explode into a mess of viscera.",
+    "category": [ "perk" ],
+    "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "OVERKILL_DAMAGE", "add": -1000.0 } ] } ]
+  },
+  {
+    "type": "mutation",
+    "id": "perk_tingly",
+    "name": { "str": "Tingly" },
+    "points": 0,
+    "description": "Ever since the apocalyse you get a funny feeling during storms and have trouble trusting small appliances.  You usually have a pretty good idea where they are though.",
```

Comment:
```suggestion
    "description": "Ever since the apocalypse you get a funny feeling during storms and have trouble trusting small appliances.  You usually have a pretty good idea where they are though.",
```